code/modules/mining/mine_items.dm

```diff
@@ -190,7 +190,7 @@
 		sleep(50)
 		var/clear = TRUE
 		for(var/turf/T in range(3,src))
```

Comment:
range 3 is larger than the actual pod
also this could be trange(which returns a list of turfs) instead

on line 197 shouldn't it be a 5x5 area?